dhctl/pkg/log/logger.go

```diff
@@ -724,7 +725,10 @@ func (d *TeeLogger) writeToFile(content string) {
 		return
 	}
 
-	if _, err := d.buf.Write([]byte(content)); err != nil {
+	timestamp := time.Now().Format(time.DateTime)
+	contentWithTimestamp := fmt.Sprintf("%s - %s", timestamp, content)
+
+	if _, err := d.buf.Write([]byte(contentWithTimestamp)); err != nil {
```

Comment:
```suggestion
	if _, err := d.buf.Write(append([]byte(time.Now().Format("2006-01-02 15:04:05 - ")), content...)); err != nil {
```

A bit more efficient version that avoids string allocation and formatting of fmt.Sprintf

Reply:
Thank you. But I thing my version more cleared for viewing. In our case some allocations have not impact for performance.